database/src/seeds/03_basic_project_survey_setup.ts

```diff
@@ -770,3 +775,31 @@ const insertAccessRequest = () => `
     $$${faker.lorem.sentences(2)}$$
   );
   `;
+
+/**
+ * SQL to insert a fake system alert
+ *
+ */
+const insertSystemAlert = () => `
+  INSERT INTO alert
+    (
+      alert_type_id,
+      name,
+      message,
+      data,
+      severity,
+      record_end_date,
+      create_user,
+      update_user
+    )
+  VALUES (
+    (SELECT alert_type_id FROM alert_type ORDER BY random() LIMIT 1),
+    $$${faker.lorem.words(3)}$$,
```

Comment:
I think none of these will show in the UI since the names wont match any of the expected ones ('Projects', etc).

Could define an array of supported names, and then pick a random one to use:

```
const SystemAlertNames = [
  'Summary',
  'Manage Telemetry',
  'Manage Observations',
  'Manage Animals',
  'Manage Sampling',
  'Projects',
  'Surveys',
  'Standards',
  'Administrator',
  'Funding'
];

const insertSystemAlert = () => `
  ...
  $$${SystemAlertNames[Math.floor(Math.random() * SystemAlertNames.length)]}$$,
  ...
`;
```

Reply:
Disregard, misread the schema!